certificates_retiring_certificates.md

```diff
@@ -1,36 +1,35 @@
 # Retiring Certificates
 
-A Retirement Account is a repository for M-RETS Certificates that the Organization wants to designate as retired for an environmental claim. 
-M-RETS Certificates in a Retirement Account are no longer transferable and serve as an electronic proof of retirement. M-RETS is not responsible for ensuring that Users retire RECs for voluntary or compliance purposes. M-RETS requires all Users to initiate and complete all retirements. Please notify M-RETS if you would like to add a new type of voluntary or compliance retirement reason. 
+M-RETS is not responsible for ensuring that Users retire RECs for voluntary or compliance purposes. M-RETS requires all Users to initiate and complete all retirements. Please notify M-RETS if you would like to add a new type of voluntary or compliance retirement reason. 
 
 ***Before retiring RECS you will need to create a retiernment account, to do so please refer [here](https://mrets.github.io/Help/add_account)***.
 
 
-### Step 1: Select the batch(es) you wish to retire from the Active Certificates table within the Certificates Dashboard. If you do not want to retire an entire batch, you will be able to edit the REC quantity from the batches later in the process.
+###  Navigate to the Certificates Dashboard and select the Active table. Select the batch(es) you wish to retire you will be able to edit the REC quantity from the batches later in the process.
```

Comment:
"Select the batch(es) you wish to retire. You will be able to edit the REC quantity from the batches later in the process."

Maybe break this into two sentences as I've shown above